markdown/bitburner.corporation.createcorporation.md

```diff
@@ -4,7 +4,7 @@
 
 ## Corporation.createCorporation() method
 
-Create a Corporation.
+Create a Corporation. You should use [canCreateCorporation](./bitburner.corporation.cancreatecorporation.md) to check if you can do it before using this function, because it throws an error in some cases.
```

Comment:
Not everyone will need to check, but it's good to mention the option.

```suggestion
Create a Corporation. You should use [canCreateCorporation](./bitburner.corporation.cancreatecorporation.md) to check if you are unsure you can do it, because it throws an error in some cases.
```